### DISCUSSION TITLE
How to run code depending on SQLAlchemy 2.0 in Dagster tasks?

### DISCUSSION THREAD

**Discussion by tokarenko** [dagster-io/dagster#19506] [Q&A] [answered]
Please advise how to run code depending on SQLAlchemy 2.0 in Dagster tasks? I am unable to run Dagster with the code in Docker container because of the following issue: ["AttributeError: 'Engine' object has no attribute 'execute'"](https://github.com/dagster-io/dagster/discussions/11881). Git workflow in Docker container reports: 
`dagster 1.3.1 requires sqlalchemy<2.0.0,>=1.0, but you have sqlalchemy 2.0.23 which is incompatible.`

> The issue seems to have resolution that works for both v1 and v2 of SQLAlchemy: https://stackoverflow.com/a/75316945

At the same time I am able to run the code depending on SQLAlchemy 2.0 in Python virtual environment with the following packages installed:
```
dagit                       1.3.10
dagster                     1.3.10
SQLAlchemy                  2.0.23
```

**Comment by alangenfeld** [comment, marked as answer]
SQLAlchemy 2.0 support was added in `1.3.8` so you need to update the version of dagster in the docker container to be that or greater.
